### ISSUE TITLE
Question about ema_cfg_time_mask and unconditional velocity field mask

### ISSUE THREAD

**Issue by Alienge** [zhuyu-cs/MeanFlow#2]
I am coming again.

I have some confusion about the following line in the code:

https://github.com/zhuyu-cs/MeanFlow/blob/a596d39aa078b66e5eb6bfe28b870d69ae6efde8/loss.py#L212

I’m not sure how cfg_min_t and cfg_max_t correspond to the parameters mentioned in the paper — could you clarify this part?

Also, in the implementation of the unconditional velocity field, the mask is used here:

https://github.com/zhuyu-cs/MeanFlow/blob/a596d39aa078b66e5eb6bfe28b870d69ae6efde8/loss.py#L235

I’m confused about the intended meaning of ema_cfg_time_mask. It seems this mask is used to determine where to apply ema_v_tilde vs. ema_v_t

Would it make more sense for this masking to be based on:
https://github.com/zhuyu-cs/MeanFlow/blob/a596d39aa078b66e5eb6bfe28b870d69ae6efde8/loss.py#L104

 If we are constructing the unconditional velocity field, shouldn't the torch.where mask correspond to dropout_mask, and the corresponding v_t should also be the unconditional one?

Looking forward to you replying!

**Comment by zhuyu-cs** [comment]
All fixed, thanks!
